### ISSUE TITLE
tt-rss syncing

### ISSUE THREAD

**Issue by brentsimmons** [brentsimmons/Evergreen#388]
tt-rss https://git.tt-rss.org/git/tt-rss/wiki/ApiReference

**Comment by translit** [comment]
Yes, please. This feature would make me consider switching back from Reeder.

**Comment by levifig** [comment]
Looking forward to this for sure… I'm sure a lot of nerds have moved to TT-RSS running on their own servers in the years following the death of Google Reader… :)

**Comment by herrbischoff** [comment]
Indeed. And given that NNW is now free and open source, supporting free and open source feed servers should be quite fitting.

**Comment by Fischmuetze** [comment]
Would be great!!!

**Comment by barolo** [comment]
Adding my vote!

**Comment by seraphyn** [comment]
You got my vote , too
Would be great addition to this feedreader 👍

**Comment by xplosionmind** [comment]
Seems like in version 6.x this implementation was discarded in favor of other services. Any plans to add it in future minor releases os should we wait for 7.x?

**Comment by seraphyn** [comment]
One Problem is the hardcoded support of feedly.
TinyTinyRSS does have a plugin to provide an api for feedly with. 
It's possible to use any reader on iOS and macOS who is using a connector for feedly.
Like unread, fiery feeds, reeder and so on.

It should be possible to set up my own address and credentials for the connection with feedly for the win 

This is how to problem could be solved

**Comment by translit** [comment]
@seraphyn 
> TinyTinyRSS does have a plugin to provide an api for feedly with.

Can you give a link to that plugin? The only one I know is the [one](https://github.com/DigitalDJ/tinytinyrss-fever-plugin) for Fever API.

**Comment by seraphyn** [comment]
Dann, you're right 😀
It's the fever plug-in.

I'm sorry 

You don't know how many times I'm getting this wrong 

Stay save and healthy

**Comment by RivenNero** [comment]
Agreed! Need this feature badly!

**Comment by vitaminx** [comment]
Saw this question three years ago, but haven't used it yet! looking forward to solving

**Comment by lmmsoft** [comment]
+1

**Comment by hnuzhoulin** [comment]
+1

**Comment by eric-pierce** [comment]
@lmmsoft @hnuzhoulin It's been several months but I just published a plugin for Tiny Tiny RSS that will allow NetNewsWire to interface with it using the FreshRSS API here: https://github.com/eric-pierce/freshapi 

I've got to say NNW is a really nice experience, better with TT-RSS as the server. If you give it a shot please provide any feedback or issues you find!

**Comment by isdamir** [comment]
> This is now possible - I just published a plugin for Tiny Tiny RSS that will allow NetNewsWire to interface with it using the FreshRSS API here: https://github.com/eric-pierce/freshapi
> 
> I've got to say NNW is a really nice experience, better with TT-RSS as the server. If you give it a shot please provide any feedback or issues you find!

Thank you very much, this has perfectly solved the problem I encountered. Once again, I express my gratitude.

**Comment by eric-pierce** [comment]
> > This is now possible - I just published a plugin for Tiny Tiny RSS that will allow NetNewsWire to interface with it using the FreshRSS API here: https://github.com/eric-pierce/freshapi
> > I've got to say NNW is a really nice experience, better with TT-RSS as the server. If you give it a shot please provide any feedback or issues you find!
> 
> Thank you very much, this has perfectly solved the problem I encountered. Once again, I express my gratitude.

Glad you're enjoying it! Let me know if you run into any issues!
